### PR TITLE
Add UI test for Capsule Content Counts

### PR DESCRIPTION
**Pull request by vsedmik**
### Problem Statement
UI coverage for Capsule Content Counts (new in 6.15) is missing.

### Solution
This PR.

### Related Issues
requires https://github.com/SatelliteQE/airgun/pull/1050

### CODE REVIEW

File: tests/foreman/ui/test_capsulecontent.py [hunk 1]
```diff
@@ -0,0 +1,237 @@
...
+                'ago' in details['content'][lce]['top_row_content']['Last sync']
+            ), 'LCE should be marked as synced'
+            assert details['content'][lce][cv.name]['mid_row_content'][
+                'Synced'
+            ], 'CV should be marked as synced'
+            repos_details = details['content'][lce][cv.name]['expanded_repo_details'][1:]
+
+            for s_repo in sat_repos:
+                c_repo = next(r for r in repos_details if r[0] == s_repo.name)
+                assert c_repo, 'Reposiory not listed'
```
Comment (synkd) [inline tests/foreman/ui/test_capsulecontent.py:177]:
```suggestion
                assert c_repo, 'Repository not listed'
```

**Review by synkd — APPROVED** [review]
ACK, with one non-blocking typo correction.

**Review by pondrejk — APPROVED** [review]
ack, pending typo

**Comment by vsedmik** [comment]
resolved typo, rebased

**Pull request merged**